rest_framework_reactive/pool.py

```diff
@@ -265,7 +272,7 @@ def stop_all(self):
         Stops all query observers.
         """
 
-        for observer in self._observers.values():
+        for observer in list(self._observers.values()):
```

Comment:
Why do you have to create a list? The for loop should iterate over the iterators.